examples/ConwaysGameOfLife/Readme.md

```diff
@@ -0,0 +1,15 @@
+## Conway's Game Of Life
+
+[The Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), also known simply as Life, is a cellular automaton devised by the British mathematician John Horton Conway in 1970.
```

Comment:
Should Life be in quotes or italicized? (I think I am being picky. Feel free to push back.)